streams/src/test/java/org/apache/kafka/streams/kstream/internals/TimeWindowedKStreamImplTest.java

```diff
@@ -158,21 +235,23 @@ public void shouldMaterializeCount() {
                 final List<KeyValue<Windowed<String>, Long>> data =
                     StreamsTestUtils.toList(windowStore.fetch("1", "2", ofEpochMilli(0), ofEpochMilli(1000L)));
 
-                assertThat(data, equalTo(Arrays.asList(
+                assertThat(data, equalTo(asList(
                     KeyValue.pair(new Windowed<>("1", new TimeWindow(0, 500)), 2L),
                     KeyValue.pair(new Windowed<>("1", new TimeWindow(500, 1000)), 1L),
-                    KeyValue.pair(new Windowed<>("2", new TimeWindow(500, 1000)), 2L))));
+                    KeyValue.pair(new Windowed<>("2", new TimeWindow(500, 1000)), 2L),
+                    KeyValue.pair(new Windowed<>("2", new TimeWindow(1000, 1500)), 1L))));
```

Comment:
Why do we get one more output record?

Reply:
I added one more input in `processData`: `inputTopic.pipeInput("2", "30", 1000L);`